overlay.nix

```diff
@@ -1,4 +1,6 @@
 self: super: {
+  jdk = super.jdk11_headless;
+
```

Comment:
Is this more idiomatic or should the overlay override maven instead ?
Passing jdk11_headless for jdk.